packages/global/src/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { jsx, type ThemeUIStyleObject } from '@theme-ui/core'
+import { css, type Theme } from '@theme-ui/css'
+import { Global as EmotionGlobal } from '@emotion/react'
+
+const Global = ({ sx }: { sx: ThemeUIStyleObject }) =>
+  jsx(EmotionGlobal, {
+    styles: (emotionTheme: unknown) => {
+      const theme = emotionTheme as Theme
+      return css(sx)(theme)
```

Comment:
I'm kinda wondering if
```ts
useMemo(() => css(sx)(theme), [sx, theme])
```
here would be an overkill.
We can always add it later after we know more I suppose.

Reply:
Oh good call. I wish evaluating perf with hooks like that with were easier…conceptually makes sense to use it at least